src/engraving/layout/layoutsystem.cpp

```diff
@@ -1219,6 +1227,43 @@ void LayoutSystem::processLines(System* system, std::vector<Spanner*> lines, boo
         }
     }
 
+    if (slurSegments.size() > 1) {
+        const qreal slurCollisionVertOffset = 0.65 * system->spatium();
```

Comment:
it's not quite clear, why you have chosen these specific number. Makes sense to have named constexpr variables

Reply:
or a comment, at least